clients/web/src/views/widgets/MetadataWidget.js

```diff
@@ -53,7 +53,7 @@ girder.views.MetadataWidget = girder.View.extend({
     initialize: function (settings) {
         this.item = settings.item;
         this.accessLevel = settings.accessLevel;
-        this.item.on('g:changed', function () {
+        this.item.on('g:fetched', function () {
```

Comment:
Those are item models, which as far as I can tell, never trigger `g:changed` (collections do though).  Models trigger `g:fetched`, and that seems to be what those listeners are intended to do.